draft-ietf-rats-corim.md

```diff
@@ -2448,8 +2449,27 @@ The comparison MUST return false if there are no hash algorithms from the condit
 
 ##### Comparison for raw-value entries
 
+A `raw-value` entry contains binary data which a Verifier following this specification will treat as opaque.
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+The value stored under `measurement-values-map` key 4 in an ACS entry must be a `raw-value` entry, which must have type `bytes`, with an optional tag.
+
+The value stored under the condition ECT `measurement-values-map` key 4 may additionally be a `raw-value-compare` entry, which specifies an expected value and an optional mask.
+
+If the condition ECT `measurement-value-map` key 4 is of type `bytes` or `tagged-bytes`, then the Verifier treats it in the same way as a `raw-value-compare` with the `value` field holding the same contents and the `mask` field omitted.
+The standard comparison function defined in this document does not use the tag value.
+
+The comparison MUST return false if the lengths of the candidate entry value and the condition ECT value are different.
+
+If a mask is provided, then the comparison MUST return false if the lengths of the condition ECT mask and value are different.
+
+If a mask is not provided, then the comparison MUST return true if and only if the condition ECT value is binary equal to the candidate entry value.
+
+If a mask field is provided, then the comparison MUST use the mask to determine which bits to compare.
+If a bit in the mask is 0 then this indicates that the corresponding bit in the ACS Entry value may have either value.
+If, for every bit position in the mask whose value is 1, the corresponding bits in both values are equal then the comparison MUST return true.
+
+For backwards compatibility, a Verifier SHOULD treat a mask provided in condition ECT `measurement-values-mask` key 5 in the same way as a mask provided in the `mask` entry of `raw-value-compare`.
```

Comment:
There's the potential for the mask in 4.raw-value-compare (or better name) to be different from the mask in 5 of both are populated. The language should be clear that if 4.raw-value-compare is used then 5 MUST/SHOULD not be used.  Or if 5 is used then 4.raw-value-compare MUST/SHOULD not be used.

SHOULD makes it harder to get verifier logic right. I'm leaning toward MUST.